packages/constants/src/constants.ts

```diff
@@ -151,5 +151,6 @@ export interface ApplicationWindow extends Window {
     enableSignUp?: boolean;
     disabledMenuItems?: string[];
     useFullRedirectsForLinks?: boolean;
+    disableInferringOfMcApiUrlOnProduction?: string;
```

Comment:
This is needed now I think as we access the property above.

Reply:
This should be a boolean

Reply:
My bad. There we go packages/constants/src/constants.ts

Reply:
My bad. There we go 31113e9